starter-code/index.js

```diff
@@ -1,11 +1,44 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
-};
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
+  }
+  get(pos) {
+    return this.items[pos - 1];
+  }
+  max() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.items[this.items.length - 1];
+    }
+  }
+  min() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.items[0];
+    }
+  }
+  average() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.items.reduce((a, b) => a + b, 0) / this.length;
```

Comment:
you can reuse your code like this:
` return this.sum() / this.length;`

because you're doing the same operation like the sum()